include/config.hpp

```diff
@@ -0,0 +1,37 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        ::::::::            */
+/*   config.hpp                                         :+:    :+:            */
+/*                                                     +:+                    */
+/*   By: lizhang <lizhang@student.codam.nl>           +#+                     */
+/*                                                   +#+                      */
+/*   Created: 2023/06/26 16:49:27 by lizhang       #+#    #+#                 */
+/*   Updated: 2023/06/26 16:49:27 by lizhang       ########   odam.nl         */
+/*                                                                            */
+/* ************************************************************************** */
+
+#ifndef CONFIG_HPP
+#define CONFIG_HPP
+
+#include <string>
+
+class Config_mac
+{
+    protected:
+        std::string	server_name;
```

Comment:
variables and functions to be named in lowerCaseCamelCase format.

Reply:
I use lowercase for variables, and use lowerCaseCamelCase format for member functions. 
And I use "_"  at the beginning of private members (ex: `int     _num_socket`).
I'm not so sure about how to use "protected" but in this case, server_name, listen_address, root_location, and index could be "private"?

Maybe we can discuss the writing rule when we meet?